### PR TITLE
Fix logging issues

### PR DESCRIPTION
**Pull request by nritsche**
- Only set handler with basicConfoig in scripts.
- Make sure sanic logging doesn't propagate to root logger.
- Disable sanic debug logging

### CODE REVIEW

File: comet/archiver.py [hunk 1]
```diff
@@ -6,22 +6,20 @@
...
 import chimedb.core as chimedb
 import chimedb.dataset as db
 
 from . import Manager, CometError, __version__
 from .broker import DEFAULT_PORT
-from .manager import TIMESTAMP_FORMAT, LOG_FORMAT
+from .manager import TIMESTAMP_FORMAT
 
 
-logging.basicConfig(format=LOG_FORMAT)
```
Comment (anjakefala) [inline comet/archiver.py]: Why did these get moved to scripts?

Comment (nritsche) [reply comet/archiver.py]: archiver.py obviously only gets used by the script anyways, but if a project imports `comet.manager` for example and it already has configured logging, it ends up with duplicate logging lines.

File: comet/broker.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 DEFAULT_PORT = 12050
 
 app = Sanic(__name__)
+logger.propagate = False
```
Comment (anjakefala) [inline comet/broker.py]: Why do we want Sanic logging to not propagate? Will it have its own handler?

File: comet/archiver.py (hunk 1 above)
Comment (anjakefala) [reply comet/archiver.py]: Gotcha!

File: comet/broker.py (hunk 2 above)
Comment (nritsche) [reply comet/broker.py]: Yeah, without this line, I get the lines from the sanic logger twice. Once in their format and once in the one configured in the scripts.

**Comment by anjakefala** [comment]
:tada:

**Pull request merged**